app/src/main/java/com/example/KnowYouLimit/dao/IDrinksDao.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.KnowYouLimit.dao
+
+import com.example.KnowYouLimit.dto.drinks
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface IDrinksDao {
+
+    @GET("")
```

Comment:
The BASE_URL will need to be populated for this to be used.  Where is it going to get the data?